extension/li3ds--1.0.0.sql

```diff
@@ -72,7 +78,11 @@ create table session(
 
 create table datasource(
     id serial primary key
-    , uri varchar
+    -- possible uri schemes are "file" and "column"
+    , uri varchar not null constraint uri_scheme check (split_part(uri, ':', 1) = any ('{file, column}'::text[]))
+    , type datasource_type not null
+    , parameters jsonb
+    , capture_time timestamptz
```

Comment:
Also, we already have `start_time` and `end_time` in the `session` table, which makes wonder about adding another time interval to the `datasource` table. Wouldn't that be redundant?